src/pages/dashboard/Drivers.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'react-router-dom'.